src/BlackSlope.Api.Common/BlackSlope.Api.Common.csproj

```diff
@@ -32,9 +36,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="4.0.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="4.0.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="4.0.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="5.0.0-rc4" />

```

Comment:
This version has been in RC mode for quite a while RC5 released a couple of days ago., I was wondering if it is safe to recommend it right now?

Reply:
Yeah they just released rc-5, so I'll upgrade to that